protos/custom.proto

```diff
@@ -16,6 +16,11 @@ service CustomService {
   // Evaluate some Lua inside of the mission and return the result as a JSON string.
   // Disabled by default.
   rpc Eval(EvalRequest) returns (EvalResponse) {}
+
+  /// Calculates the magnetic declination at the given position using the International Geomagnetic
+  /// Reference Field (IGRF) model. The result is not always exactly the same as what DCS seem to
+  /// use, but it is very close (DCS doesn't expose its declination).
```

Comment:
Does `///` work as a doc comment for the doc generator?

Reply:
I don't think so.
```c
/*
 *
 */
```
is the multiline format.